# string motor_name
int32 val

# success
int32 SUCCESS=1
int32 FAILURE=0
int32 NONE = -1

# shutdown errors
int32 OVERLOAD_ERROR=100
int32 ELECTRICAL_SHOCK_ERROR=101
int32 MOTOR_ENCODER_ERROR=102
int32 OVERHEATING_ERROR=103
int32 INPUT_VOLTAGE_ERROR=104

int32 COMMUNICATION_ERROR = 200
